app/client/src/components/editorComponents/ActionCreator/index.tsx

```diff
@@ -361,6 +376,12 @@ function getFieldFromValue(
       field: FieldType.CLEAR_INTERVAL_ID_FIELD,
     });
   }
+
+  if (value.indexOf("getGeolocation") !== -1) {
```

Comment:
```suggestion
  if (value.indexOf("getCurrentPosition") !== -1) {
```
I noticed the callBack field doesn't show up during action creation. This might be the reason?